chain/consensus/hierarchical/subnet/manager/manager.go

```diff
@@ -530,7 +567,7 @@ func (s *SubnetMgr) LeaveSubnet(
 
 	// See if we are already syncing with that chain. If this
 	// is the case we can remove the subnet
-	if sh, _ := s.getSubnet(id); s != nil {
+	if sh, _ := s.getSubnet(id); sh != nil {
```

Comment:
That's good. I just wanted to report it but you have already fixed this one too.